eventmesh-connectors/eventmesh-connector-jdbc/src/main/java/org/apache/eventmesh/connector/jdbc/connection/JdbcConnection.java

```diff
@@ -432,13 +442,11 @@ public <T> T preparedQuery(String sql, ResultSetMapper<T> resultSetMapper, Prepa
      * @throws SQLException if a database access error occurs.
      */
     public <T> T preparedQuery(String sql, PreparedStatementFactory preparedStatementFactory, ResultSetMapper<T> resultSetMapper,
-        PreparedParameter... preparedParameters) throws SQLException {
+                               PreparedParameter... preparedParameters) throws SQLException {
 
```

Comment:
Redundant indention here. We rarely use this kind of line break. There are other similar places.